sdk/rust-examples/file-seek/src/main.rs

```diff
@@ -0,0 +1,41 @@
+//! File Seeker Example
+//!
+//! ## Context
+//!
+//! Read a file, move the cursor forward for a number of bytes and write the remaining bytes into
+//! another text file
+//!
+//! ## Authors
+//!
+//! The Veracruz Development Team.
+//!
+//! ## Copyright
+//!
+//! See the file `LICENSING.markdown` in the Veracruz root directory for licensing and
+//! copyright information.
+
+use std::fs::File;
+use std::io;
+use std::io::prelude::*;
+use std::io::SeekFrom;
+
+fn main() -> io::Result<()> {
+    let mut f = File::open("/input/rust-std.txt")?;
```

Comment:
Can you float these hardcoded filenames out as top-level constants?

```rust
const INPUT_FILENAME: &'static str = "/input/rust-std.txt";
```

...or similar.